components/layout.tsx

```diff
@@ -58,24 +63,33 @@ const TopNav: React.FC = () => {
     <>
       <Navbar
         bg="secondary"
+        sticky={isMobile ? "top" : undefined}
         variant="dark"
         expand={false}
         expanded={isExpanded}
       >
         <Container>
-          <Navbar.Toggle aria-controls="topnav" onClick={toggleNav} />
-          <Navbar.Brand>
-            <Nav.Link href="/" className={`py-0`}>
-              <Image fluid src="nav-logo.png" alt="logo"></Image>
-            </Nav.Link>
-          </Navbar.Brand>
-          <Nav>
-            {!authenticated ? (
-              <SignInWithModal />
-            ) : (
-              <ProfileLink displayName={profile?.displayName} />
-            )}
-          </Nav>
+          <div className={styles.navbar_boxes_container}>
+            <div className={styles.navbar_box}>
+              <Navbar.Toggle aria-controls="topnav" onClick={toggleNav} />
+            </div>
+
+            <Navbar.Brand>
+              <Nav.Link href="/" className="py-0">
+                <Image fluid src="nav-logo.png" alt="logo"></Image>
+              </Nav.Link>
+            </Navbar.Brand>
+
+            <div className={styles.navbar_box}>
+              <Nav>
+                {authenticated ? (
+                  <ProfileLink displayName={profile?.displayName} />
```

Comment:
This thing has a lot of padding around it -- I think we can remove the margin [here](https://github.com/codeforboston/advocacy-maps/blob/868028230084f759546cc6dea556280dfdf01e3f/components/ProfileLink/ProfileLink.js#L21)

Reply:
We can do this only for screens smaller than 992px because when the profile name becomes visible and margin is 0, there's no spacing between image and name.

<img width="940" alt="image" src="https://user-images.githubusercontent.com/25043084/184274673-55c2377b-67d1-4b12-98fb-30e202685ac4.png">

Reply:
Great catch!